pkg/controller/util.go

```diff
@@ -82,6 +84,19 @@ func getSecretName(client kubernetes.Interface, pvc *v1.PersistentVolumeClaim) (
 	return name, nil
 }
 
+// returns the PVC size with fs overhead percentage included.
+func getPvcSize(pvc *v1.PersistentVolumeClaim) (string, error) {
```

Comment:
This needs to be called something different.  I'd read 'getPvcSize' simply as "get me the size of the specified pvc", that's not what this is.  Maybe "getFilesystemAdjustedSize" or something?

Reply:
sure. maybe simply "getAdjustedPvcSize"?